src/All-Movies/All-Movies.js

```diff
@@ -3,7 +3,6 @@ import MovieCard from '../Movie-Card/Movie-Card'
 import './All-Movies.css'
 
 const AllMovies = ({movies, switchView}) => {
-    console.log(movies)
     const moviePosters = movies.map(movie => {
         return (
             <MovieCard
```

Comment:
If you wanted to pass less props, you could send thru just `movie` and destructure on the MovieCard component - totally up to you tho!